sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/MeterSharedState.java

```diff
@@ -91,7 +91,7 @@ public final WriteableMetricStorage registerSynchronousMetricStorage(
       try {
         storage.add(getMetricStorageRegistry().register(currentStorage));
       } catch (DuplicateMetricStorageException e) {
-        logger.log(Level.WARNING, e, () -> "Failed to register metric.");
+        logger.log(Level.WARNING, DebugUtils.duplicateMetricErrorMessage(e), e);
```

Comment:
using the original form (with the message Supplier) here might be a little better, in case someone doesn't care and turns this up to ERROR only.